recipes/large-image-converter/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "large-image-converter" %}
+{% set version = "1.8.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3a132f2c722f54dbb772e3e7a020bbe38a6876642071544e3a677807f42b8f4a
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  # [win, py <= 36]
```

Comment:
3.6 should be allowed (`< 36`, not `<= 36`)

Reply:
Mondays.....